src/components/manage/Blocks/Hero/schema.js

```diff
@@ -13,17 +13,25 @@ const messages = defineMessages({
     id: 'Link to',
     defaultMessage: 'Link to',
   },
+  Align: {
+    id: 'Alignment',
+    defaultMessage: 'Alignment',
+  },
 });
 
-const schemaHero = ({ intl }) => {
+const schemaHero = ({ intl, variation }) => {
   return {
     title: 'Block settings',
     required: [],
     fieldsets: [
       {
         id: 'default',
         title: intl.formatMessage(messages.more),
-        fields: ['linkTitle', 'linkHref'],
+        fields: [
```

Comment:
@danielamormocea We also need a Image Sizes widget, the current image renders smaller than those in image Block.

Reply:
For the Background Image template, I could also imagine a `opacity` slider to alter the transparency for bg image....🤔